2024-01-06 00:55:03,224 T0000: NOTICE WORKER = gw7
2024-01-06 00:55:03,224 T0000: NOTICE using testbed file /data/work/logs/docker-master-448374-l3/testbed_1_1_4.yaml
2024-01-06 00:55:03,225 T0000: NOTICE Execution Start Time: 2024-01-06 00:55:03.221328
2024-01-06 00:55:03,225 T0000: NOTICE LOGS PATH: /data/work/logs/docker-master-448374-l3/gw7
2024-01-06 00:55:03,796 T0000: NOTICE VERSION: UNKNOWN UNKNOWN UNKNOWN
2024-01-06 00:55:03,797 T0000: NOTICE HOST: sonicmgmtmaster MEM: 376gb 53.2%
2024-01-06 00:55:03,797 T0000: NOTICE ROOT_PATH: /repo
2024-01-06 00:55:03,797 T0000: NOTICE Python: 3.8.10
2024-01-06 00:55:03,797 T0000: NOTICE Topology unknown: D1T1:4
2024-01-06 00:55:03,797 T0000: DEBUG 
SUITE ARGS: --load-image none --max-time module 9000 --max-time function 5400 --max-time session 3600 --logs-level=debug --topology-check=function --env SPYTEST_TESTBED_IGNORE_CONSTRAINTS TG,TGCARD,TGSPEED,CHIP,MODEL --noop --env SPYTEST_RESET_CONSOLES 0 --noop --env SPYTEST_ONREBOOT_RENEW_MGMT_IP 1 --noop --env SPYTEST_RECOVERY_CR_PER_CMD_POST_OP 1 --noop --env SPYTEST_BATCH_POLL_STATUS_TIME 300 --noop --env SPYTEST_BATCH_DEAD_NODE_MAX_TIME 7200 --noop --breakout-mode none --env SPYTEST_SHUTDOWN_FREE_PORTS 1 --env SPYTEST_HOOKS_BREAKOUT_UITYPE click --env SPYTEST_ONREBOOT_RENEW_MGMT_IP 0 --env SPYTEST_VERIFY_CSV_MIN_TOPOLOGY 1 --env SPYTEST_CMD_FAIL_RESULT_SUPPORT 0 --env SPYTEST_ROUTING_CONFIG_MODE split --env SPYTEST_CLEAR_DEVICE_METADATA_BGP_ASN 1 --env SPYTEST_TEXTFSM_USE_BUNDLED_PARSER 1 --feature-group master --ifname-type native --feature-disable bcmcmd --feature-disable warm-reboot --save config-db module --tclist-csv TestBGPConfed.test_confed_route_distribution_with_rmap,TestBGPConfed.test_ipv6_confed_route_distribution,TestBGPConfed.test_ipv6_confed_with_rr,TestBGPIPvxRouteAdvertisementFilter.test_bgp_ebgp4_nbr_authentication,TestBGPIPvxRouteAdvertisementFilter.test_bgp_ebgp4_nbr_update_source,TestBGPIPvxRouteAdvertisementFilter.test_bgp_route_map_with_community,TestBGPIPvxRouteAdvertisementFilter.test_default_originate_ipv4,TestBGPIPvxRouteAdvertisementFilter.test_distribute_list_in_ipv4,TestBGPIPvxRouteAdvertisementFilter.test_distribute_list_in_ipv6,TestBGPIPvxRouteAdvertisementFilter.test_filter_list_in_ipv4,TestBGPIPvxRouteAdvertisementFilter.test_filter_list_in_ipv6,TestBGPIPvxRouteAdvertisementFilter.test_filter_list_out_ipv6,TestBGPIPvxRouteAdvertisementFilter.test_prefix_list_out_ipv4,TestBGPIPvxRouteAdvertisementFilter.test_prefix_list_out_ipv6,TestBGPIPvxRouteAdvertisementFilter.test_route_aggregate_ipv6,TestBGPIPvxRouteAdvertisementFilter.test_route_map_in_ipv4,TestBGPIPvxRouteAdvertisementFilter.test_route_map_in_ipv6,TestBGPIPvxRouteAdvertisementFilter.test_static_blackhole_rt_redistribute_with_routemap_ipv6,TestBGPL3Lag.test_ft_bgp_l3lag_peer_traffic_check,TestBGPRif.test_bgp_route_aggregation_4byteASN,TestBGPRif.test_ft_bgp_clear,TestBGPRif.test_ft_bgp_graceful_restart_and_aware_routers,TestBGPRif.test_ft_bgp_peer_traffic_check,TestBGPRif.test_ft_bgp_rmap,TestBGPRif.test_ft_bgp_v4_dyn_nbr,TestBGPRif.test_ft_bgp_v6_dyn_nbr,TestBGPRif.test_ft_bgp_v6_link_local_bgp,TestBGPRrTraffic.test_ft_bgp6_rr_traffic_check,TestBGPVeLag.test_ft_bgp_clear,TestBGPVeLag.test_ft_bgp_peer_traffic_check,test_VrfFun001_06,test_VrfFun_05_50,test_VrfFun_10_12_14,test_VrfFun_20_24_25_32_33_44_45,test_VrfFun_26_27,test_cli_validation_bgp_router_config,test_cli_validation_ip_address,test_ft_arp_dynamic_renew_traffic_test,test_ft_bgp_ebgp_community_map,test_ft_bgp_ibgp_RR_Loop,test_ft_ip6_static_route_traffic_forward_blackhole,test_ft_ip_static_ip_on_mgmt_intrf,test_ft_ip_static_route_traffic_forward,test_ft_ip_v4_v6_L2_L3_translation,test_ft_ipv6_neighbor_entry,test_ft_l3_performance_enhancements_v4_route_intstall_withdraw,test_ft_nat_docker_restart,test_ft_ping_v4_v6_after_ip_change_pc,test_ft_ping_v4_v6_vlan,test_ft_qos_config_mgmt_verifying_config_with_save_reboot,test_ft_sflow_max_sflow_collector_config,test_ft_sflow_sampling_v6_sFlow_collector,test_ft_system_config_mgmt_verifying_config_with_save_reboot_sflow,test_ft_verify_interfaces_order,test_l2_to_l3_port,test_l3_v4_route_po_1,test_vrf_bgp,test_vrf_reload,test_vrf_route_leak,test_vrf_scale,TestBGPRif.test_ft_bgp_ipv4_no_route_aggregation_for_exact_prefix_match,TestBGPRif.test_ft_bgp_ipv4_route_aggregation_atomic_aggregate_without_as_set,TestBGPRif.test_ft_bgp_ipv6_route_aggregation_with_as_set,TestBGPRif.test_ft_bgp_v4_max_dyn_nbr,TestBGPIPvxRouteAdvertisementFilter.test_redistribute_connected_ipv4,TestBGPIPvxRouteAdvertisementFilter.test_redistribute_static_ipv4,TestBGPIPvxRouteAdvertisementFilter.test_redistribute_connected_ipv6,TestBGPIPvxRouteAdvertisementFilter.test_redistribute_static_ipv6,TestBGPIPvxRouteAdvertisementFilter.test_default_originate_ipv6,test_spytest_infra_first,test_spytest_infra_second,test_spytest_infra_last --noop

2024-01-06 00:55:03,797 T0000: NOTICE ARGS: --logs-level debug --results-prefix=results --get-tech-support none --fetch-core-files none --syslog-check none --save config-db module --topology-check module --logs-path logs/2024-01-06-00-50-16 --continue-on-collection-errors --max-time session 0 --max-time module 0 --max-time function 0 --noop --tryssh=0 --testbed-file testbed.yaml --env SPYTEST_RPS_DEBUG 0 --tclist-bucket 1,2,4 --test-suite community-vs-l3 --email rama.kristipati@broadcom.com --env SPYTEST_SCAPY_DBG_LVL 2 --noop --env SPYTEST_EMAIL_FROM SPyTest-VSNet@broadcom.com --noop --logs-path logs/docker-master-448374-l3 --env SPYTEST_EMAIL_BODY_PREFIX http://10.193.81.56:8080/sonic_mgmt_master/work/logs/docker-master-448374-l3/dashboard.html --email-subject-nargs VSNet-sonic_mgmt_master-DOCKER-L3-{{bldver}} : {{passrate}}: {{pass_cnt}}/{{executed}} docker-master-448374-l3 --noop --feature-group master
2024-01-06 00:55:04,524 T0000: DEBUG 'errors' not set in properties for V3 assuming 'default'
2024-01-06 00:55:04,525 T0000: INFO  TGEN-IP 127.0.0.1
2024-01-06 00:55:04,525 T0000: WARN  Registering Topology
2024-01-06 00:55:04,541 T0000: INFO  Connecting all devices
2024-01-06 00:55:04,556 T0000: INFO  Loading scapy:1.0 127.0.0.1 Ports: ['1/9', '1/10', '1/11', '1/12'] skip: False
2024-01-06 00:55:04,822 T0002: INFO  [D1-V3] Connecting to device (V3): sonic_ssh: 172.18.0.4:22 ..
2024-01-06 00:55:04,823 T0002: INFO  [D1-V3] TRY Authentication: 172.18.0.4:22 admin YourPaSsWoRd
2024-01-06 00:55:05,585 T0000: INFO  TG Scapy Init
2024-01-06 00:55:05,586 T0000: INFO  TG Base Init skip: False
2024-01-06 00:55:05,598 T0000: INFO  TG Scapy Connect 127.0.0.1:8009
2024-01-06 00:55:05,719 T0000: INFO  Server Logs Path /data/work/scapy-tgen/logs/current/inst-2024_01_06_00_55_05_632366/gw7/default.log
2024-01-06 00:55:05,725 T0000: DEBUG REQ: tg_connect(port_list=['1/9', '1/10', '1/11', '1/12'])
2024-01-06 00:55:10,050 T0002: DEBUG [D1-V3] Connected ...
2024-01-06 00:55:10,872 T0002: INFO  [D1-V3] Prompt at the connection start: 'admin@V3:~$' 
2024-01-06 00:55:10,873 T0002: AUDIT [D1-V3] ['export TMOUT=0', 'stty cols 5000', 'uptime']
2024-01-06 00:55:11,281 T0002: INFO  [D1-V3] FCMD: export TMOUT=0;stty cols 5000;uptime
2024-01-06 00:55:11,542 T0002: INFO  [D1-V3]  00:55:11 up 29 days,  2:00,  1 user,  load average: 9.85, 12.87, 11.23
2024-01-06 00:55:11,542 T0002: INFO  [D1-V3] admin@V3:~$ 
2024-01-06 00:55:11,556 T0002: AUDIT [D1-V3] cat /proc/uptime
2024-01-06 00:55:11,936 T0000: DEBUG RESP: tg_connect {"port_handle": {"1/9": "port-9", "1/10": "port-10", "1/11": "port-11", "1/12": "port-12"}, "status": 1}
2024-01-06 00:55:11,944 T0000: INFO  TG Connection: Success
2024-01-06 00:55:11,944 T0000: INFO  TG Connect...done
2024-01-06 00:55:11,961 T0002: INFO  [D1-V3] FCMD: cat /proc/uptime
2024-01-06 00:55:12,217 T0002: INFO  [D1-V3] 2512834.07 198068911.49
2024-01-06 00:55:12,217 T0002: INFO  [D1-V3] admin@V3:~$ 
2024-01-06 00:55:12,220 T0002: INFO  [D1-V3] UPTIME: 2512834.07
2024-01-06 00:55:12,220 T0002: AUDIT [D1-V3] sudo vtysh -c 'show running-config | include hostname'
2024-01-06 00:55:12,627 T0002: INFO  [D1-V3] FCMD: sudo vtysh -c 'show running-config | include hostname'
2024-01-06 00:55:13,287 T0002: INFO  [D1-V3] Error response from daemon: No such container: bgp
2024-01-06 00:55:13,287 T0002: INFO  [D1-V3] admin@V3:~$ 
2024-01-06 00:55:13,289 T0002: WARN  [D1-V3] Skipped error checking in session init but detected pattern: containing_no_response_from_container command: sudo vtysh -c 'show running-config | include hostname'
2024-01-06 00:55:13,290 T0002: WARN  [D1-V3] Failed to read hostname from vtysh retry again in 5 sec
2024-01-06 00:55:13,290 T0002: INFO  Sleep for 5 sec(s)... @371
2024-01-06 00:55:18,296 T0002: AUDIT [D1-V3] sudo vtysh -c 'show running-config | include hostname'
2024-01-06 00:55:18,705 T0002: INFO  [D1-V3] FCMD: sudo vtysh -c 'show running-config | include hostname'
2024-01-06 00:55:19,520 T0002: INFO  [D1-V3] Error response from daemon: No such container: bgp
2024-01-06 00:55:19,521 T0002: INFO  [D1-V3] admin@V3:~$ 
2024-01-06 00:55:19,524 T0002: WARN  [D1-V3] Skipped error checking in session init but detected pattern: containing_no_response_from_container command: sudo vtysh -c 'show running-config | include hostname'
2024-01-06 00:55:19,526 T0002: WARN  [D1-V3] Failed to read hostname from vtysh retry again in 5 sec
2024-01-06 00:55:19,526 T0002: INFO  Sleep for 5 sec(s)... @371
2024-01-06 00:55:24,528 T0002: AUDIT [D1-V3] sudo vtysh -c 'show running-config | include hostname'
2024-01-06 00:55:24,935 T0002: INFO  [D1-V3] FCMD: sudo vtysh -c 'show running-config | include hostname'
2024-01-06 00:55:25,649 T0002: INFO  [D1-V3] Error response from daemon: No such container: bgp
2024-01-06 00:55:25,649 T0002: INFO  [D1-V3] admin@V3:~$ 
2024-01-06 00:55:25,652 T0002: WARN  [D1-V3] Skipped error checking in session init but detected pattern: containing_no_response_from_container command: sudo vtysh -c 'show running-config | include hostname'
2024-01-06 00:55:25,654 T0002: WARN  [D1-V3] Failed to read hostname from vtysh retry again in 5 sec
2024-01-06 00:55:25,655 T0002: INFO  Sleep for 5 sec(s)... @371
2024-01-06 00:55:30,660 T0002: AUDIT [D1-V3] sudo vtysh -c 'show running-config | include hostname'
2024-01-06 00:55:31,067 T0002: INFO  [D1-V3] FCMD: sudo vtysh -c 'show running-config | include hostname'
2024-01-06 00:55:31,777 T0002: INFO  [D1-V3] Error response from daemon: No such container: bgp
2024-01-06 00:55:31,777 T0002: INFO  [D1-V3] admin@V3:~$ 
2024-01-06 00:55:31,780 T0002: WARN  [D1-V3] Skipped error checking in session init but detected pattern: containing_no_response_from_container command: sudo vtysh -c 'show running-config | include hostname'
2024-01-06 00:55:31,781 T0002: WARN  [D1-V3] Failed to read hostname from vtysh retry again in 5 sec
2024-01-06 00:55:31,782 T0002: INFO  Sleep for 5 sec(s)... @371
2024-01-06 00:55:36,784 T0002: AUDIT [D1-V3] sudo vtysh -c 'show running-config | include hostname'
2024-01-06 00:55:37,192 T0002: INFO  [D1-V3] FCMD: sudo vtysh -c 'show running-config | include hostname'
2024-01-06 00:55:37,956 T0002: INFO  [D1-V3] Error response from daemon: No such container: bgp
2024-01-06 00:55:37,956 T0002: INFO  [D1-V3] admin@V3:~$ 
2024-01-06 00:55:37,958 T0002: WARN  [D1-V3] Skipped error checking in session init but detected pattern: containing_no_response_from_container command: sudo vtysh -c 'show running-config | include hostname'
2024-01-06 00:55:37,960 T0002: WARN  [D1-V3] Failed to read hostname from vtysh retry again in 5 sec
2024-01-06 00:55:37,960 T0002: INFO  Sleep for 5 sec(s)... @371
2024-01-06 00:55:42,964 T0002: AUDIT [D1-V3] sudo vtysh -c 'show running-config | include hostname'
2024-01-06 00:55:43,369 T0002: INFO  [D1-V3] FCMD: sudo vtysh -c 'show running-config | include hostname'
2024-01-06 00:55:44,080 T0002: INFO  [D1-V3] Error response from daemon: No such container: bgp
2024-01-06 00:55:44,081 T0002: INFO  [D1-V3] admin@V3:~$ 
2024-01-06 00:55:44,083 T0002: WARN  [D1-V3] Skipped error checking in session init but detected pattern: containing_no_response_from_container command: sudo vtysh -c 'show running-config | include hostname'
2024-01-06 00:55:44,086 T0002: WARN  [D1-V3] Failed to read hostname from vtysh retry again in 5 sec
2024-01-06 00:55:44,086 T0002: INFO  Sleep for 5 sec(s)... @371
2024-01-06 00:55:49,092 T0002: AUDIT [D1-V3] sudo vtysh -c 'show running-config | include hostname'
2024-01-06 00:55:49,500 T0002: INFO  [D1-V3] FCMD: sudo vtysh -c 'show running-config | include hostname'
2024-01-06 00:55:50,164 T0002: INFO  [D1-V3] Error response from daemon: No such container: bgp
2024-01-06 00:55:50,165 T0002: INFO  [D1-V3] admin@V3:~$ 
2024-01-06 00:55:50,169 T0002: WARN  [D1-V3] Skipped error checking in session init but detected pattern: containing_no_response_from_container command: sudo vtysh -c 'show running-config | include hostname'
2024-01-06 00:55:50,172 T0002: WARN  [D1-V3] Failed to read hostname from vtysh retry again in 5 sec
2024-01-06 00:55:50,172 T0002: INFO  Sleep for 5 sec(s)... @371
2024-01-06 00:55:55,178 T0002: AUDIT [D1-V3] sudo vtysh -c 'show running-config | include hostname'
2024-01-06 00:55:55,586 T0002: INFO  [D1-V3] FCMD: sudo vtysh -c 'show running-config | include hostname'
2024-01-06 00:55:56,298 T0002: INFO  [D1-V3] Error response from daemon: No such container: bgp
2024-01-06 00:55:56,299 T0002: INFO  [D1-V3] admin@V3:~$ 
2024-01-06 00:55:56,301 T0002: WARN  [D1-V3] Skipped error checking in session init but detected pattern: containing_no_response_from_container command: sudo vtysh -c 'show running-config | include hostname'
2024-01-06 00:55:56,302 T0002: WARN  [D1-V3] Failed to read hostname from vtysh retry again in 5 sec
2024-01-06 00:55:56,302 T0002: INFO  Sleep for 5 sec(s)... @371
2024-01-06 00:56:01,304 T0002: AUDIT [D1-V3] sudo vtysh -c 'show running-config | include hostname'
2024-01-06 00:56:01,711 T0002: INFO  [D1-V3] FCMD: sudo vtysh -c 'show running-config | include hostname'
2024-01-06 00:56:02,375 T0002: INFO  [D1-V3] Error response from daemon: No such container: bgp
2024-01-06 00:56:02,375 T0002: INFO  [D1-V3] admin@V3:~$ 
2024-01-06 00:56:02,378 T0002: WARN  [D1-V3] Skipped error checking in session init but detected pattern: containing_no_response_from_container command: sudo vtysh -c 'show running-config | include hostname'
2024-01-06 00:56:02,379 T0002: WARN  [D1-V3] Failed to read hostname from vtysh retry again in 5 sec
2024-01-06 00:56:02,379 T0002: INFO  Sleep for 5 sec(s)... @371
2024-01-06 00:56:07,384 T0002: AUDIT [D1-V3] sudo vtysh -c 'show running-config | include hostname'
2024-01-06 00:56:07,793 T0002: INFO  [D1-V3] FCMD: sudo vtysh -c 'show running-config | include hostname'
2024-01-06 00:56:08,454 T0002: INFO  [D1-V3] Error response from daemon: No such container: bgp
2024-01-06 00:56:08,454 T0002: INFO  [D1-V3] admin@V3:~$ 
2024-01-06 00:56:08,457 T0002: WARN  [D1-V3] Skipped error checking in session init but detected pattern: containing_no_response_from_container command: sudo vtysh -c 'show running-config | include hostname'
2024-01-06 00:56:08,459 T0002: WARN  [D1-V3] Failed to read hostname from vtysh retry again in 5 sec
2024-01-06 00:56:08,459 T0002: INFO  Sleep for 5 sec(s)... @371
2024-01-06 00:56:13,464 T0002: AUDIT [D1-V3] sudo vtysh -c 'show running-config | include hostname'
2024-01-06 00:56:13,870 T0002: INFO  [D1-V3] FCMD: sudo vtysh -c 'show running-config | include hostname'
2024-01-06 00:56:14,578 T0002: INFO  [D1-V3] Error response from daemon: No such container: bgp
2024-01-06 00:56:14,578 T0002: INFO  [D1-V3] admin@V3:~$ 
2024-01-06 00:56:14,580 T0002: WARN  [D1-V3] Skipped error checking in session init but detected pattern: containing_no_response_from_container command: sudo vtysh -c 'show running-config | include hostname'
2024-01-06 00:56:14,581 T0002: WARN  [D1-V3] Failed to read hostname from vtysh retry again in 5 sec
2024-01-06 00:56:14,581 T0002: INFO  Sleep for 5 sec(s)... @371
2024-01-06 00:56:19,584 T0002: AUDIT [D1-V3] sudo vtysh -c 'show running-config | include hostname'
2024-01-06 00:56:19,993 T0002: INFO  [D1-V3] FCMD: sudo vtysh -c 'show running-config | include hostname'
2024-01-06 00:56:20,654 T0002: INFO  [D1-V3] Error response from daemon: No such container: bgp
2024-01-06 00:56:20,654 T0002: INFO  [D1-V3] admin@V3:~$ 
2024-01-06 00:56:20,657 T0002: WARN  [D1-V3] Skipped error checking in session init but detected pattern: containing_no_response_from_container command: sudo vtysh -c 'show running-config | include hostname'
2024-01-06 00:56:20,659 T0002: WARN  [D1-V3] Failed to read hostname from vtysh retry again in 5 sec
2024-01-06 00:56:20,659 T0002: INFO  Sleep for 5 sec(s)... @371
2024-01-06 00:56:25,664 T0002: AUDIT [D1-V3] sudo vtysh -c 'show running-config | include hostname'
2024-01-06 00:56:26,073 T0002: INFO  [D1-V3] FCMD: sudo vtysh -c 'show running-config | include hostname'
2024-01-06 00:56:26,835 T0002: INFO  [D1-V3] Error response from daemon: No such container: bgp
2024-01-06 00:56:26,835 T0002: INFO  [D1-V3] admin@V3:~$ 
2024-01-06 00:56:26,837 T0002: WARN  [D1-V3] Skipped error checking in session init but detected pattern: containing_no_response_from_container command: sudo vtysh -c 'show running-config | include hostname'
2024-01-06 00:56:26,838 T0002: WARN  [D1-V3] Failed to read hostname from vtysh retry again in 5 sec
2024-01-06 00:56:26,838 T0002: INFO  Sleep for 5 sec(s)... @371
2024-01-06 00:56:31,844 T0002: AUDIT [D1-V3] sudo vtysh -c 'show running-config | include hostname'
2024-01-06 00:56:32,250 T0002: INFO  [D1-V3] FCMD: sudo vtysh -c 'show running-config | include hostname'
2024-01-06 00:56:32,960 T0002: INFO  [D1-V3] Error response from daemon: No such container: bgp
2024-01-06 00:56:32,961 T0002: INFO  [D1-V3] admin@V3:~$ 
2024-01-06 00:56:32,964 T0002: WARN  [D1-V3] Skipped error checking in session init but detected pattern: containing_no_response_from_container command: sudo vtysh -c 'show running-config | include hostname'
2024-01-06 00:56:32,966 T0002: WARN  [D1-V3] Failed to read hostname from vtysh retry again in 5 sec
2024-01-06 00:56:32,966 T0002: INFO  Sleep for 5 sec(s)... @371
2024-01-06 00:56:37,972 T0002: AUDIT [D1-V3] sudo vtysh -c 'show running-config | include hostname'
2024-01-06 00:56:38,379 T0002: INFO  [D1-V3] FCMD: sudo vtysh -c 'show running-config | include hostname'
2024-01-06 00:56:39,142 T0002: INFO  [D1-V3] Error response from daemon: No such container: bgp
2024-01-06 00:56:39,142 T0002: INFO  [D1-V3] admin@V3:~$ 
2024-01-06 00:56:39,145 T0002: WARN  [D1-V3] Skipped error checking in session init but detected pattern: containing_no_response_from_container command: sudo vtysh -c 'show running-config | include hostname'
2024-01-06 00:56:39,148 T0002: WARN  [D1-V3] Failed to read hostname from vtysh retry again in 5 sec
2024-01-06 00:56:39,148 T0002: INFO  Sleep for 5 sec(s)... @371
2024-01-06 00:56:44,152 T0002: AUDIT [D1-V3] sudo vtysh -c 'show running-config | include hostname'
2024-01-06 00:56:44,560 T0002: INFO  [D1-V3] FCMD: sudo vtysh -c 'show running-config | include hostname'
2024-01-06 00:56:45,473 T0002: INFO  [D1-V3] Error response from daemon: No such container: bgp
2024-01-06 00:56:45,473 T0002: INFO  [D1-V3] admin@V3:~$ 
2024-01-06 00:56:45,475 T0002: WARN  [D1-V3] Skipped error checking in session init but detected pattern: containing_no_response_from_container command: sudo vtysh -c 'show running-config | include hostname'
2024-01-06 00:56:45,476 T0002: WARN  [D1-V3] Failed to read hostname from vtysh retry again in 5 sec
2024-01-06 00:56:45,477 T0002: INFO  Sleep for 5 sec(s)... @371
2024-01-06 00:56:50,480 T0002: AUDIT [D1-V3] sudo vtysh -c 'show running-config | include hostname'
2024-01-06 00:56:50,889 T0002: INFO  [D1-V3] FCMD: sudo vtysh -c 'show running-config | include hostname'
2024-01-06 00:56:51,751 T0002: INFO  [D1-V3] Error response from daemon: No such container: bgp
2024-01-06 00:56:51,751 T0002: INFO  [D1-V3] admin@V3:~$ 
2024-01-06 00:56:51,753 T0002: WARN  [D1-V3] Skipped error checking in session init but detected pattern: containing_no_response_from_container command: sudo vtysh -c 'show running-config | include hostname'
2024-01-06 00:56:51,754 T0002: WARN  [D1-V3] Failed to read hostname from vtysh retry again in 5 sec
2024-01-06 00:56:51,754 T0002: INFO  Sleep for 5 sec(s)... @371
2024-01-06 00:56:56,760 T0002: AUDIT [D1-V3] sudo vtysh -c 'show running-config | include hostname'
2024-01-06 00:56:57,167 T0002: INFO  [D1-V3] FCMD: sudo vtysh -c 'show running-config | include hostname'
2024-01-06 00:56:57,980 T0002: INFO  [D1-V3] Error response from daemon: No such container: bgp
2024-01-06 00:56:57,980 T0002: INFO  [D1-V3] admin@V3:~$ 
2024-01-06 00:56:57,982 T0002: WARN  [D1-V3] Skipped error checking in session init but detected pattern: containing_no_response_from_container command: sudo vtysh -c 'show running-config | include hostname'
2024-01-06 00:56:57,984 T0002: WARN  [D1-V3] Failed to read hostname from vtysh retry again in 5 sec
2024-01-06 00:56:57,985 T0002: INFO  Sleep for 5 sec(s)... @371
2024-01-06 00:57:02,988 T0002: AUDIT [D1-V3] sudo vtysh -c 'show running-config | include hostname'
2024-01-06 00:57:03,396 T0002: INFO  [D1-V3] FCMD: sudo vtysh -c 'show running-config | include hostname'
2024-01-06 00:57:04,208 T0002: INFO  [D1-V3] Error response from daemon: No such container: bgp
2024-01-06 00:57:04,208 T0002: INFO  [D1-V3] admin@V3:~$ 
2024-01-06 00:57:04,210 T0002: WARN  [D1-V3] Skipped error checking in session init but detected pattern: containing_no_response_from_container command: sudo vtysh -c 'show running-config | include hostname'
2024-01-06 00:57:04,211 T0002: WARN  [D1-V3] Failed to read hostname from vtysh retry again in 5 sec
2024-01-06 00:57:04,211 T0002: INFO  Sleep for 5 sec(s)... @371
2024-01-06 00:57:09,216 T0002: AUDIT [D1-V3] sudo vtysh -c 'show running-config | include hostname'
2024-01-06 00:57:09,621 T0002: INFO  [D1-V3] FCMD: sudo vtysh -c 'show running-config | include hostname'
2024-01-06 00:57:10,381 T0002: INFO  [D1-V3] Error response from daemon: No such container: bgp
2024-01-06 00:57:10,382 T0002: INFO  [D1-V3] admin@V3:~$ 
2024-01-06 00:57:10,384 T0002: WARN  [D1-V3] Skipped error checking in session init but detected pattern: containing_no_response_from_container command: sudo vtysh -c 'show running-config | include hostname'
2024-01-06 00:57:10,385 T0002: WARN  [D1-V3] Failed to read hostname from vtysh retry again in 5 sec
2024-01-06 00:57:10,385 T0002: INFO  Sleep for 5 sec(s)... @371
2024-01-06 00:57:15,388 T0002: ERROR [D1-V3] Failed to read hostname from vtysh - assuming sonic
2024-01-06 00:57:15,390 T0002: INFO  [D1-V3] Base Prompt: sonic
2024-01-06 00:57:15,390 T0002: DEBUG [D1-V3] Wait for system ready Ref: 866
2024-01-06 00:57:15,797 T0002: INFO  [D1-V3] FCMD: sudo md5sum /etc/spytest/remote/spytest-helper.py
2024-01-06 00:57:16,057 T0002: INFO  [D1-V3] 4522abaafb8e29890a8377a810228525  /etc/spytest/remote/spytest-helper.py
2024-01-06 00:57:16,057 T0002: INFO  [D1-V3] admin@V3:~$ 
2024-01-06 00:57:16,063 T0002: WARN  [D1-V3] Execute 'sudo python /etc/spytest/remote/spytest-helper.py --wait-for-ports 300' with timeout 600 sec -- TRY 1
2024-01-06 00:57:16,064 T0002: INFO  [D1-V3] SCMD: sudo python /etc/spytest/remote/spytest-helper.py --wait-for-ports 300  
2024-01-06 00:57:19,889 T0002: WARN  [D1-V3] Skipped error checking in session init but detected pattern: containing_python_traceback command: sudo python /etc/spytest/remote/spytest-helper.py --wait-for-ports 300  
2024-01-06 00:57:19,891 T0002: INFO  [D1-V3] 
2024-01-06 00:57:19,891 T0002: INFO  [D1-V3] Remote CMD: 'show platform summary'
2024-01-06 00:57:19,891 T0002: INFO  [D1-V3] Error: Failed to execute 'show platform summary' (out: '') (err: 'Traceback (most recent call last):
2024-01-06 00:57:19,891 T0002: INFO  [D1-V3]   File "/usr/local/bin/show", line 5, in <module>
2024-01-06 00:57:19,891 T0002: INFO  [D1-V3]     from show.main import cli
2024-01-06 00:57:19,892 T0002: INFO  [D1-V3]   File "/usr/local/lib/python3.11/dist-packages/show/main.py", line 307, in <module>
2024-01-06 00:57:19,892 T0002: INFO  [D1-V3]     if is_gearbox_configured():
2024-01-06 00:57:19,892 T0002: INFO  [D1-V3]        ^^^^^^^^^^^^^^^^^^^^^^^
2024-01-06 00:57:19,892 T0002: INFO  [D1-V3]   File "/usr/local/lib/python3.11/dist-packages/show/main.py", line 248, in is_gearbox_configured
2024-01-06 00:57:19,892 T0002: INFO  [D1-V3]     app_db.connect(app_db.APPL_DB)
2024-01-06 00:57:19,892 T0002: INFO  [D1-V3]   File "/usr/lib/python3/dist-packages/swsscommon/swsscommon.py", line 1821, in connect
2024-01-06 00:57:19,892 T0002: INFO  [D1-V3]     return _swsscommon.SonicV2Connector_Native_connect(self, db_name, retry_on)
2024-01-06 00:57:19,892 T0002: INFO  [D1-V3]            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
2024-01-06 00:57:19,893 T0002: INFO  [D1-V3] RuntimeError: Unable to connect to redis: Cannot assign requested address')
2024-01-06 00:57:19,893 T0002: INFO  [D1-V3] 
2024-01-06 00:57:19,893 T0002: INFO  [D1-V3] Remote CMD: 'cat /usr/share/sonic/device/None/None/port_config.ini | tail -1'
2024-01-06 00:57:19,893 T0002: INFO  [D1-V3] Remote CMD: 'sudo show interfaces status '
2024-01-06 00:57:19,893 T0002: INFO  [D1-V3] Error: Failed to execute 'sudo show interfaces status ' (out: '') (err: 'Traceback (most recent call last):
2024-01-06 00:57:19,893 T0002: INFO  [D1-V3]   File "/usr/local/bin/show", line 5, in <module>
2024-01-06 00:57:19,893 T0002: INFO  [D1-V3]     from show.main import cli
2024-01-06 00:57:19,893 T0002: INFO  [D1-V3]   File "/usr/local/lib/python3.11/dist-packages/show/main.py", line 307, in <module>
2024-01-06 00:57:19,894 T0002: INFO  [D1-V3]     if is_gearbox_configured():
2024-01-06 00:57:19,894 T0002: INFO  [D1-V3]        ^^^^^^^^^^^^^^^^^^^^^^^
2024-01-06 00:57:19,894 T0002: INFO  [D1-V3]   File "/usr/local/lib/python3.11/dist-packages/show/main.py", line 248, in is_gearbox_configured
2024-01-06 00:57:19,894 T0002: INFO  [D1-V3]     app_db.connect(app_db.APPL_DB)
2024-01-06 00:57:19,894 T0002: INFO  [D1-V3]   File "/usr/lib/python3/dist-packages/swsscommon/swsscommon.py", line 1821, in connect
2024-01-06 00:57:19,894 T0002: INFO  [D1-V3]     return _swsscommon.SonicV2Connector_Native_connect(self, db_name, retry_on)
2024-01-06 00:57:19,894 T0002: INFO  [D1-V3]            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
2024-01-06 00:57:19,895 T0002: INFO  [D1-V3] RuntimeError: Unable to connect to redis: Cannot assign requested address')
2024-01-06 00:57:19,895 T0002: INFO  [D1-V3] 
2024-01-06 00:57:19,895 T0002: INFO  [D1-V3] admin@V3:~$ 
2024-01-06 00:57:19,896 T0002: ERROR [D1-V3] Failed to execute the command sudo python /etc/spytest/remote/spytest-helper.py --wait-for-ports 300  
2024-01-06 00:57:19,896 T0002: ERROR exception in thread: Traceback (most recent call last):
  File "/repo/utilities/parallel.py", line 159, in _thread_func
    retvals[index] = func(*args, **kwargs)
  File "/repo/spytest/net.py", line 1064, in connect_to_device
    show_ver_output = self.connect_to_device_real(access, retry, recon, recover)
  File "/repo/spytest/net.py", line 1290, in connect_to_device_real
    self.do_common_init(devname, phase=0, max_ready_wait=max_ready_wait)
  File "/repo/spytest/net.py", line 866, in do_common_init
    sys_status = self.wa.wait_system_status(devname, max_time=max_ready_wait)
  File "/repo/spytest/framework.py", line 1616, in wait_system_status
    return self._wait_for_ports(dut, max_time)
  File "/repo/spytest/framework.py", line 1605, in _wait_for_ports
    return self.net.apply_remote(dut, "wait-for-ports", [max_time])
  File "/repo/spytest/net.py", line 4523, in apply_remote
    raise ValueError(msg)
ValueError: Failed to execute the command sudo python /etc/spytest/remote/spytest-helper.py --wait-for-ports 300  

2024-01-06 00:57:19,897 T0002: ERROR [D1-V3] Failed to connect to device
2024-01-06 00:57:19,897 T0000: ERROR Failed to connect (['V3'])
2024-01-06 00:57:19,897 T0000: INFO  --FINISH--RUN--4 @559
